src/pages/ReceiverDetails.tsx

```diff
@@ -116,6 +159,41 @@ export const ReceiverDetails = () => {
     } as React.CSSProperties;
   };
 
+  const renderRetryInvitationButton = () => {
+    return (
+      <Button
+        variant="secondary"
+        size="xs"
+        onClick={(e) => {
+          e.preventDefault();
+          retryReceiverInvitation();
+        }}
+        isLoading={isInvitationRetryFetching}
+        disabled={Boolean(selectedWallet?.stellarAddress)}
+        {...(selectedWallet?.stellarAddress
+          ? { title: "This wallet has already been registered" }
+          : {})}
+      >
+        Retry invitation message
+      </Button>
+    );
+  };
+
+  const renderUnregisterWalletButton = () => {
+    return (
+      <Button
+        variant="secondary"
+        size="xs"
+        onClick={showUnregisterModal}
+        isLoading={isUnregisterWalletPending}
+        disabled={isUnregisterWalletPending}
```

Comment:
If the trigger is the same, we don't need both `isLoading` and `disabled` props. `isLoading` is enough.